.github/workflows/ci.yml

```
@@ -19,13 +19,13 @@ env:
 
 jobs:
   pre-commit:
-    name: Pre-commit checks
+    name: Pre-commit Checks
     uses: ./.github/workflows/pre-commit-run.yml
     with:
-      pre-commit-source: "pre-commit"
+      pre-commit-source: pre-commit
 
   test-docs-build-verify:
-    name: Test Verify Docs Build
+    name: Verify Docs Build Check
```

Minor Nit: The task is verifying the docs build; verifying a check is... a bit meta.
```suggestion
    name: Verify Docs Build
```